src/components/data-display/tables/DomainsTable.tsx

```diff
@@ -114,22 +128,36 @@ const DomainsTable = ({
 
       Object.entries(domainData.names).map(([domain, record]) => {
         const ant = domainData.ants[record.processId];
+        const isIoCompatible =
+          walletAddress && ant?.state && record?.processId
+            ? !doAntsRequireUpdate({
+                ants: {
+                  [record.processId]: {
+                    state: ant.state,
+                    handlers: ant.handlers,
+                  },
+                },
+                userAddress: walletAddress.toString(),
+              })
```

Comment:
this function looks like it still checks lua source code tx id vs. just validating the handlers exist on the ANT - can you confirm?

Reply:
Nope - this is checking that 1, the ant can be updated by the current user, and 2, if the handlers match those listed in the SDK